package.json

```diff
@@ -27,8 +27,8 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "expect": "^1.20.2",
     "mocha-multi": "^1.0.1",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
Upgrading to have access to React Hooks (state, ref, etc.) Class components are kind-of an opt-in thing now.